### PR TITLE
Fix documentation code formatting

### PR DESCRIPTION
**Pull request by regisb**
Code blocks should always follow an empty line; otherwise, jekyll will
fail to properly format the code block. Also, github's code fencing (with triple backticks) does not work in the middle of item lists.

### CODE REVIEW

File: docs/State-checkpointing.md [hunk 1]
```diff
@@ -22,31 +22,30 @@ For example a word count bolt could use the key value state abstraction for the
...
-       Integer count = wordCounts.get(word, 0);
-       count++;
-       wordCounts.put(word, count);
-       collector.emit(tuple, new Values(word, count));
-       collector.ack(tuple);
-     }
- ...
- }
- ```
+    public class WordCountBolt extends BaseStatefulBolt<KeyValueState<String, Long>> {
```
Comment (HeartSaVioR) [inline docs/State-checkpointing.md]: Maybe missed wrapping to code block.

**Comment by HeartSaVioR** [comment]
@regisb Looks good overall without minor. Left comment.

File: docs/State-checkpointing.md (hunk 1 above)
Comment (HeartSaVioR) [inline docs/State-checkpointing.md]: Ah never mind. I got what you mean that `code fencing (with triple backticks) does not work in the middle of item lists`.

**Comment by HeartSaVioR** [comment]
+1

**Comment by HeartSaVioR** [comment]
Thanks @regisb, merged into master and 1.x-branch.

I've had other works to do, so website update will be done later.

**Comment by regisb** [comment]
Great, thanks for the quick review and merge!

**Pull request merged**